(DOCSP-39501): Consolidate Quick Start page (#3280)
mongodb/docs-realm: source/includes/api-details/csharp/quick-start/quick-start-import-realm-description.rst

> Add the following line to the top of your source files to use the SDK:

Reviewer: [suggestion] unless I'm missing something, this can probably be made into a generic includes to use in several SDKs (C#, JS, TS, Swift, Kotlin)
```suggestion
Add the following to the top any source files that use the SDK:
```
